lib/old_archive_test.go

```diff
@@ -93,14 +93,14 @@ func TestOldArchive(t *testing.T) {
 			metadata := `{"filename": "` + filename + `", "options": {}}`
 			fs := makeMemMapFs(t, map[string][]byte{
 				// files
-				"/files/github.com/loadimpact/k6/samples/example.js": []byte(`github file`),
+				"/files/github.com/k6io/k6/samples/example.js": []byte(`github file`),
```

Comment:
Fix the indentation in this file please

Reply:
I was really hoping those will be caught by the golanci-lint, but apparently because there were other `gofumpt` reports it just decided this is not new :facepalm:

Reply:
fixed in 5bb4f49 along others